server.js

```diff
@@ -101,12 +197,9 @@ let createToken = user => {
   }, signature, { expiresIn: '7d' });
 }
 
+
 let signIn = (request, response) => {
-  let body = '';
-  request.on('data', (chunk) => {
-    body += chunk.toString();
-  });
-  request.on('end', () => {
+    readIncoming(request, (incoming) => {
```

Comment:
Thanks for putting this into a function! I was going to eventually refactor it so i appreciate you taking the time to do it :)